src/annotator/integrations/test/html-metadata-test.js

```diff
@@ -177,83 +70,218 @@ describe('HTMLMetadata', () => {
-      ]);
-    });
+      it('should return Highwire metadata', () => {
+        assert.ok(metadata.highwire);
+        assert.deepEqual(metadata.highwire.pdf_url, ['foo.pdf']);
+        assert.deepEqual(metadata.highwire.doi, ['10.1175/JCLI-D-11-00015.1']);
+        assert.deepEqual(metadata.highwire.title, ['Foo']);
+      });
 
-    it('should return PRISM metadata', () => {
-      assert.ok(metadata.prism);
-      assert.deepEqual(metadata.prism.title, ['Literary Machines']);
-    });
+      it('should return Dublin Core metadata', () => {
+        assert.ok(metadata.dc);
+        assert.deepEqual(metadata.dc.identifier, [
+          'doi:10.1175/JCLI-D-11-00015.1',
+          'foobar-abcxyz',
+        ]);
+        assert.deepEqual(metadata.dc['relation.ispartof'], ['isbn:123456789']);
+        assert.deepEqual(metadata.dc.type, ['Article']);
+      });
 
-    it('should return Twitter card metadata', () => {
-      assert.ok(metadata.twitter);
-      assert.deepEqual(metadata.twitter.site, ['@okfn']);
-    });
+      it('should return Facebook metadata', () => {
+        assert.ok(metadata.facebook);
+        assert.deepEqual(metadata.facebook.url, ['http://example.com']);
+      });
 
-    it('should return favicon URL', () => {
-      assert.equal(metadata.favicon, 'http://example.com/images/icon.ico');
-    });
+      it('should return eprints metadata', () => {
+        assert.ok(metadata.eprints);
+        assert.deepEqual(metadata.eprints.title, [
+          'Computer Lib / Dream Machines',
+        ]);
+      });
 
-    it('should set `documentFingerprint` to the dc resource identifiers URN href', () => {
-      assert.equal(metadata.documentFingerprint, metadata.link[9].href);
-    });
+      it('should return PRISM metadata', () => {
+        assert.ok(metadata.prism);
+        assert.deepEqual(metadata.prism.title, ['Literary Machines']);
+      });
 
-    it('should ignore `<link>` tags with invalid URIs', () => {
-      tempDocumentHead.innerHTML = `
-        <link rel="alternate" href="https://example.com/foo">
-        <link rel="alternate" href="http://a:b:c">
-      `;
-
-      const metadata = testDocument.getDocumentMetadata();
-
-      // There should be one link with the document location and one for the
-      // valid `<link>` tag.
-      assert.deepEqual(metadata.link.length, 2);
-      assert.deepEqual(metadata.link[1], {
-        rel: 'alternate',
-        href: 'https://example.com/foo',
-        type: '',
+      it('should return Twitter card metadata', () => {
+        assert.ok(metadata.twitter);
+        assert.deepEqual(metadata.twitter.site, ['@okfn']);
       });
-    });
 
-    it('should ignore favicons with invalid URIs', () => {
-      tempDocumentHead.innerHTML = `
-        <link rel="favicon" href="http://a:b:c">
-      `;
-      const metadata = testDocument.getDocumentMetadata();
-      assert.isUndefined(metadata.favicon);
+      it('should ignore `pdf_url` links with invalid URIs', () => {
```

Comment:
This test is about the `metadata.link` property, but is not grouped with the other tests related to link parsing.